packages/blocto-sdk/src/providers/ethereum.ts

```diff
@@ -163,7 +163,8 @@ export default class EthereumProvider
           name,
           display_name,
           network_type,
-          wallet_web_url: this._blocto.walletServer,
+          wallet_web_url: this.injectedWalletServer ||
+          ETH_ENV_WALLET_SERVER_MAPPING[blocto_service_environment],
```

Comment:
when executing this method for the first time, `this._blocto.walletServer` is an empty string.
use `this.injectedWalletServer || ETH_ENV_WALLET_SERVER_MAPPING[blocto_service_environment]` to avoid setting an empty string for the first time.

Reply:
Maybe extract `this.injectedWalletServer || ETH_ENV_WALLET_SERVER_MAPPING[blocto_service_environment]` as a constant, and we can just reuse this constant in L155 and L166? This can provide a clearer expression for the relation here, which help people maintaining the code to understand the code better